finality-aleph/src/network/mod.rs

```diff
@@ -25,8 +25,13 @@ use component::{
 };
 use manager::SessionCommand;
 
-pub use aleph::NetworkData as AlephNetworkData;
+pub use aleph::{NetworkData as AlephNetworkData, NetworkWrapper};
```

Comment:
At the top of this file – remove the comment and the permission for dead code. Fix everything that breaks as the result of this change ofc. :P

Reply:
Done